rootfs/etc/nginx/lua/configuration.lua

```diff
@@ -26,11 +41,17 @@ function _M.call()
     return
   end
 
-  ngx.req.read_body()
+  local backends = _M.new_backends()
+  if not backends then
+    ngx.log(ngx.ERR, "dynamic-configuration: POST carries empty response body")
```

Comment:
> dynamic-configuration

I'm not sure what value this adds. Can you describe a real use case?

Reply:
Other users might see the error, and see that it originated from this line in configuration.lua, but still might not know what "component" of ingress-nginx is not working. Given the annotation is named `dynamic-configuration`, I figured that this is a good way of making error logs from this lua module more meaningful.